### PR TITLE
Types translation

### PR DESCRIPTION
**Pull request by Mazur-Sebastian**

### CODE REVIEW

File: src/views/Supply.ts [hunk 1]
```diff
@@ -103,4 +103,29 @@ export namespace Supply {
         type: PrintType;
         quantity: number;
     }
+
+    export const typeName = (type: string) => {
```
Comment (grzegorz-bielski) [inline src/views/Supply.ts]: We should create a translate function for each type.

**Review by lukaszmindpal — APPROVED** [review]

File: src/views/Supply.ts (hunk 1 above)
Comment (Mazur-Sebastian) [reply src/views/Supply.ts]: I will implement i18 when the page will be released

Comment (ghost) [reply src/views/Supply.ts]: We are still lacking the table in the "i can help" flow, so there is some time left

File: src/translation/translation.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+                disposable: "Jednorazowe",
+                reusable: "Wielorazowe",
+                Male: "Męskie",
+                Female: "Damskie",
+                Child: "Dziecięce",
+                XS: "XS",
+                S: "S",
+                M: "M",
+                L: "L",
+                XL: "XL"
```
Comment (grzegorz-bielski) [inline src/translation/translation.ts:18]: It's ok for now, but we really shouldn't mix these types under one scope.

**Pull request merged**